src/main/java/hardcorequesting/team/PlayerEntry.java

```diff
@@ -118,7 +119,6 @@ public void write(JsonWriter out) throws IOException {
         out.endObject();
     }
 
-    @SideOnly(Side.SERVER)
```

Comment:
Then you need to fix the code used on item pickup and not remove an annotation that is put there for that exact reason. Now that pickup code will only work in SP and will still crash in MP.